src/app/api/user/route.ts

```diff
@@ -105,7 +112,7 @@ export const GET = async (request: NextRequest) => {
   }
   try {
     const fetchedUser = await prisma.user.findUnique({
-      where: { id },
+      where: id ? { id } : { email },
```

Comment:
This makes typescript a little angry -- as there are linter issues with the building.